resource/labeling-app/app.js

```diff
@@ -42,7 +43,7 @@ export default new Vue({
         isCurrentItemLabeled() {
             const annotation = this.annotation;
             if (this.type === 'image-object') {
-                return annotation?.label?.filter(e => e.label).length > 0;
+                return annotation?.label?.filter(e => e.label && config.categories[e.label]).length > 0;
```

Comment:
There's a corner case when a user has an empty category key, for example:
![image](https://user-images.githubusercontent.com/7889985/100088728-de7e1b80-2e50-11eb-9aa0-d71245d01461.png)

![image](https://user-images.githubusercontent.com/7889985/100088704-d4f4b380-2e50-11eb-99af-6c4c3fd58da2.png)

it's not yet supported since the next button won't be active anyway, but I think we could do a little better in the current fix by checking `e.label  in config.categories` instead of `config.categories[e.label]`

Reply:
Used a variable that will replace empty key. Fixed in e72c6f0